Stockwatch/Stockwatch.Businesses/StockSymbolService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Stockwatch.Model;
+
+namespace Stockwatch.Business
+{
+    public interface IStockSymbolService
+    {
+        void AddStock(StockSymbol stocksymbol);
+        StockSymbol FetchstockwId(int id);
+        StockSymbol FetchstockwName(string Name);
```

Comment:
Similar comment as above

Reply:
Changed to FetchStockSymbolById and FetchStockSymbolByName